sebastianbergmann/phpunit
# Add missing global assertion functions

I use the `Functions.php` in most of my projects for brevity (and to avoid discussions regarding `Assert::`, `self::`, `$this->`), but noticed that `assertEqualsWithDelta` is missing. A quick grep confirmed that it's not the only missing function. 

Since I'm not sure if you want to add the other functions I just wrote a lazy test which ensures that functions are not forgotten in the future.

Open questions:
 - not sure where you'd like the test to be, but I got confused by all the stuff in the `end-to-end` dir
 - test that documentation is up to date?
 - remove documentation and add `@see` to `Assert::` methods
 - run all tests from `AssertTest.php` against the mapped functions just to be safe?
